desidiff/nb/desidiff.ipynb

```diff
@@ -146,7 +147,9 @@
     "        difmask = {key: newmask[key] + refmask[key]\n",
     "                       for key in newmask.keys()}\n",
     "        difwave = dict(newwave)\n",
-    "\n",
+    "        \n",
+    "        redshift = z_data[0][1] #it seems each set only has one value\n",
```

Comment:
What is this doing?

Reply:
This gets the redshift data which I pulled from the z_best files. It's a little dumb right now, but every iteration only has 1 targetID, and so z_data is a length 1 array, containing a tuple with TARGETID, Z, ZERR, ZWARN, and SPECTYPE. z_data[0][1] is just the redshift for that targetID, which I need to shift the spectra into the rest frame